index.ts

```diff
@@ -60,7 +60,30 @@ async function main() {
   await downloadCSV(buildPath(dayAfter(lastImportedDate + 'T00:00:00'), yesterday), yesterday)
   console.log('csv downloaded')
   console.log('importing CSV into postgres')
-  await runCommand(formatPSQLBash(`\\copy scarf_events_raw FROM './${yesterday}.csv' WITH (FORMAT CSV, HEADER true)`))
+  const fields = [
+    "id",
+    "type",
+    "package",
+    "version",
+    "time timestamp",
```

Comment:
should this just be "time"?

```suggestion
    "time",
```